Detections/ThreatIntelligenceIndicator/DomainEntity_PaloAlto.yaml

```diff
@@ -17,37 +17,41 @@ triggerThreshold: 0
 tactics:
   - Impact
 query: |

```

Comment:
There should be an empty line after query: |

Reply:
We should make sure this requirement is detailed in the style guide.